package.json

```diff
@@ -109,6 +106,12 @@
     "semver": "^7.3.5",
     "source-map-support": "^0.5.19"
   },
+  "peerDependencies": {
+    "@grpc/grpc-js": "^1.3.2",
+    "@huddly/camera-proto": "^1.0.2",
+    "@huddly/camera-switch-proto": "^0.0.1",
```

Comment:
Could you bump the versions of these two in this PR?
camera-proto              => 1.0.4
camera-switch-proto   =>   0.0.2